go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -21,7 +21,9 @@ class GoModUpdater
           # (Private) module could not be found
           /cannot find module providing package/.freeze,
           # Package in module was likely renamed or removed
-          /module .* found \(.*\), but does not contain package/m.freeze
+          /module .* found \(.*\), but does not contain package/m.freeze,
+          # Package does not exist or has been pulled
+          /go: .*: unknown revision/m.freeze
```

Comment:
I think you also get this error if you're unauthenticated and the `go` tool can't get the module (either from Git directly or Go module proxy). Not sure if you want to update the comment in the code to make it clear.

Reply:
Thanks, added a note to clarify that 👍